package.json

```diff
@@ -19,6 +19,8 @@
   "scripts": {
     "build": "lerna run build",
     "lint": "lerna run lint",
+    "api": "lerna run api",
+    "watch": "lerna run --paralel watch",
```

Comment:
Shouldn't this be a double-l parallel? 😅

Reply:
fixed